infrastructure/alerts.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  alert_resource_group_name = "wa-${var.env}"
+}
+
+module "wa-task-disposal-action-group" {
+  source                 = "git@github.com:hmcts/cnp-module-action-group"
+  location               = "global"
+  env                    = var.env
+  resourcegroup_name     = local.alert_resource_group_name
+  action_group_name      = "${var.application_name}-${var.env}-ag"
+  short_name             = "dispr-alert"
+  email_receiver_name    = "WA Task Deletion Failure Alert"
+  email_receiver_address = data.azurerm_key_vault_secret.wa-support-email.value
+}
+
+module "task-deletion-failure-alert" {
```

Comment:
In fact we have put all our alerts in our wa-shared-infrastructure repo to keep them in once place.  Can you move this to there please?
https://github.com/hmcts/wa-shared-infrastructure/blob/master/alerts.tf

There are many wa-support alerts which span across the product and not just task-management-api so wanted them in once place.